tests/unittests/OperatorTest.cpp

```diff
@@ -10710,7 +10739,10 @@ TEST_P(OperatorTest, DivSizeT) {
   auto *result = F_->createSave("save", R);
   bindings_.allocate(result->getPlaceholder());
 
-  EE_.compile(CompilationMode::Infer);
+  CompilationContext cctx;
+  cctx.compMode = CompilationMode::Infer;
+  cctx.optimizationOpts.enableTypeDemotion = false;
```

Comment:
Also curious why this is necessary here?

Reply:
The test specifically tests div of Int64ITy/size_t. So disabling optimization to preserve test intention.